addons/knowledge/views/knowledge_article_views.xml

```diff
@@ -84,12 +84,11 @@
                                                 <i class="rounded-circle text-center fa fa-comments" title="Open chatter"/>
                                             </a>
                                             <div class="o_knowledge_share_panel dropdown-menu" role="menu">
-                                                <div class="px-3 pb-2">
-                                                    <button type="action" name="%(knowledge_invite_action_from_article)d"
-                                                            class="btn btn-primary w-100"
-                                                            attrs="{'invisible': [('user_has_write_access', '=', False)]}">
-                                                        <i class="fa fa-plus"/> Invite
-                                                    </button>
+                                                <div class="o_knowledge_share_panel_invite input-group o_cursor_pointer px-3 pb-2" attrs="{'invisible': [('user_has_write_access', '=', False)]}">
+                                                    <div class="form-control">Add people or email addresses</div>
+                                                    <div class="input-group-append">
+                                                        <button class="btn btn-primary">Invite</button>
+                                                    </div>
```

Comment:
could be done without any added js by doing : 
```
<button type="action" name="%(knowledge_invite_action_from_article)d"
             class="input-group o_cursor_pointer px-3 pb-2"
             attrs="{'invisible': [('user_has_write_access', '=', False)]}">
    <div class="form-control text-left">Add people or email addresses...</div>
    <div class="input-group-append">
         <div class="btn btn-primary">Invite</div>
    </div>
 </button>
```

Reply:
It does not work with Bootstrap 5 :cry: To call the action without listener, we need to use the button tag and this tag introduces many css rules that we would have to be overwrite to get the result we want. For the share panel rework, I will need to define that listener to intercept the click event and prevent the share panel from closing. So, I think it's okay to introduce it here.